opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1059,8 +1066,8 @@ namespace Opm
             // update the segment pressure
             {
                 const int sign = dwells[seg][SPres] > 0.? 1 : -1;
-                const double dx_limited = sign * std::min(std::abs(dwells[seg][SPres]), relaxation_factor * max_pressure_change);
-                // const double dx_limited = sign * std::min(std::abs(dwells[seg][SPres]) * relaxation_factor, max_pressure_change);
+                //const double dx_limited = sign * std::min(std::abs(dwells[seg][SPres]), relaxation_factor * max_pressure_change);
+                const double dx_limited = sign * std::min(std::abs(dwells[seg][SPres]) * relaxation_factor, max_pressure_change);
```

Comment:
This looks like a pure bugfix? Is there any reason to keep the out-commented line?

Reply:
No I don't think so. The out-comment lines where kept for the other cases. I can remove them all.

Reply:
It is not a bug. At the old time, first version MSW that ran through model 2, the original code helped us to go through difficult cases.  Now, a lot of things are improved, situation can be different.